doc/analyze-from-tfs.md

```diff
@@ -140,6 +140,9 @@ Try modifying the build definition to remove the SonarQube.MSBuild.Runner.exe en
 
 Most analysis-related configuration or execution errors will cause the build to fail and will be appear on the Build Summary. Additional information can be found by viewing the logs or diagnostic information (i.e. by clicking on **View Log**, or **Diagnostics** at the top of the Build Summary page).
 
+#### During a XAML build, the analysis fails with a "System.ArgumentNullException ... Parameter name: tfsUri"
+
+Make sure you are using at least TFS 2013 Update 2 agent software. 

```

Comment:
As far as I can tell, there is only one version of the TFS 2013 OM (https://visualstudiogallery.msdn.microsoft.com/3278bfa7-64a7-4a75-b0da-ec4ccb8d21b6). I've tested on TFS 2015 Agents + this OM in the past.